src/mito/README.md

```diff
@@ -0,0 +1,3 @@
+# Mito
+
+Mito is GreptimeDB's default table engine.
```

Comment:
Jeremy Clarkson:  "you can't be a true petrolhead until you've owned an Alfa Romeo".

Reply:
> Jeremy Clarkson: "you can't be a true petrolhead until you've owned an Alfa Romeo".

But you've already sold it 🤣